frontend/src/components/common/NotificationMessage.vue

```diff
@@ -25,7 +22,11 @@ const props = defineProps({
             :severity="props.severity"
             :sticky="true"
         >
-            <Icon :icon="props.icon" medium />
+            <Icon
+                icon="checkmark--filled"
+                :size=IconSize.medium
+                :class="'custom-carbon-icon-checkmark--filled'"
```

Comment:
Hi Jota, if we need to do the style override for this icon, then we don't need to give a class. We only give custom class if we need custom style. Also even we need a custom class, it shouldn't have both "", and ''

Reply:
ok, will do

Reply:
Yes, agree, pass argument only when it is needed, don't confuse other developers for no reason.
I just have question, where do all these "custom-carbon-icon-" class coming from? I don't seem to see them in our frontend project. Why did we define them? Are all default carbon icons need to be "customed"?

Reply:
We don't need these. We just need to define one custom icon class for one icon to override its color, that's it. So here we need a cleanup to remove all unnecessary ones.